test/cql-pytest/test_using_timeout.py

```diff
@@ -152,3 +153,19 @@ def invalid(stmt):
     invalid(f"SELECT * FROM {table} USING TIMEOUT 60s AND TTL 10000")
     invalid(f"SELECT * FROM {table} USING TIMEOUT 60s AND TTL 123 AND TIMESTAMP 911")
     invalid (f"DELETE FROM {table} USING TIMEOUT 60s AND TTL 42 WHERE p = 42")
+
+def test_truncate_using_timeout(scylla_only, cql, table1):
+    table = table1
+    key = unique_key_int()
+    cql.execute(f"INSERT INTO {table} (p,c,v) VALUES ({key},1,1)")
+    res = list(cql.execute(f"SELECT ttl(v), writetime(v) FROM {table} WHERE p = {key} and c = 1"))
```

Comment:
selecting ttl(v), writetime(v) in this test instead of just v is weird ;-) But it's not wrong and I guess you copied it from a different test.

Reply:
cleaned up in v3